vividus/src/main/java/org/vividus/configuration/ConfigurationResolver.java

```diff
@@ -59,6 +61,11 @@
     private static final String PLACEHOLDER_SUFFIX = "}";
     private static final String PLACEHOLDER_VALUE_SEPARATOR = "=";
 
+    private static final String[] DEFAULTS_PATHS = {
+        "org/vividus/http/client",
+        "org/vividus/util"
+    };
```

Comment:
it's better to define full resource paths:
```suggestion
    private static final String[] DEFAULTS_PATHS = {
        "org/vividus/http/client/defaults.properties",
        "org/vividus/util/defaults.properties"
    };
```